jetty-server/src/main/java/org/eclipse/jetty/server/MultiPartFormInputStream.java

```diff
@@ -185,12 +186,16 @@ protected void write(byte[] bytes, int offset, int length) throws IOException
         @Override
         public void write(String fileName) throws IOException
         {
+            Path p = Path.of(fileName);
```

Comment:
Note. `Path.of(String...)` is new for Java 11.
The original technique would be `Paths.get(String...)`